src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveDefendantResponseTest.java

```diff
@@ -60,6 +61,7 @@ public void shouldReturnNewlyCreatedDefendantResponse() throws Exception {
             .contains(Optional.of(response));
     }
 
+    @Ignore
```

Comment:
Are you planning to enable that test back later?

Reply:
Yes